backend/src/repository/user.repo.ts

```diff
@@ -0,0 +1,19 @@
+import { Repository, Sequelize } from "sequelize-typescript";
+import { User } from "../model/user.model";
+
+class UserRepo {
+  private userRepo: Repository<User>;
+  constructor(db: Sequelize) {
+    this.userRepo = db.getRepository(User);
```

Comment:
i see no need for repository. is they a reason you introduced it?

Reply:
I used repo here so that it doesn't become tightly coupled to Sequelize. Just in case we want to change the ORM, we can easily do that